plugins/woocommerce/src/Internal/StockNotifications/Functions.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Automattic\WooCommerce\Internal\StockNotifications;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Functions class for stock notifications.
+ */
+class Functions {
+	/**
+	 * Returns verification codes expiration time threshold (in seconds).
+	 *
+	 * @return int
+	 */
+	public static function get_verification_expiration_time_threshold() {
+		return (int) apply_filters( 'woocommerce_bis_verification_expiration_time_threshold', HOUR_IN_SECONDS );
```

Comment:
Using the same filter name as the extension because I'm assuming we are aiming to support folks migrating from the extension.